producer/spec/service_consumers/pact_helper.rb

```diff
@@ -7,13 +7,13 @@
       Cat.create!(name: 'Felix', age: 99)
       Cat.create!(name: 'Sylvester', age: 76)
     end
-
-    tear_down do
-      Cat.destroy_all
-    end
   end
 end
 
+Pact.tear_down do
```

Comment:
I found out the hard way that Pact doesn't do anything to tear down (or transaction) things for you. Rather than pulling in a dependency like database_cleaner at this point, just delete the things always.